src/styles/utils/function.scss

```diff
@@ -0,0 +1,30 @@
+@import "variables.scss";
```

Comment:
you can import it in main style file, so variables will be visible everywhere

Reply:
It doesn't work this way. If I remove @import "variables.scss"; from src/styles/utils/function.scss, and add it to main.scss, the file doesn't compile